ARPG/Assets/Scripts/PlayerStats.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour, IDamageable
+{
+    //hp variables
+    public float baseMaxHealth = 100f;
+    public float maxHealthModPerc = 1f;
+    public float MaxHealth => baseMaxHealth * maxHealthModPerc;
+
+    
+    public float currentHealth; //for clamp functionality
```

Comment:
Does this need to be public? Consider making it private and renaming it accordingly.

Reply:
both this and currentmana was made into public variables after it being explicitly requested that all variables be public in favor of ease of use in the unity editor. 
any and all fields be privatized at a later point or when a proper decision has been made in the group regarding what should be public and what should not, but for now i will be leaving them public.

Reply:
Why do you want it public?

If a variable doesn't need to be changed in testing or editing, having it be private usually makes the inspector easier to read. If having it be private causes errors, then have it be public. 

"Any and all fields" will not be privatezed at a later point. This is sure to cause errors due to different scripts access levels to others fields.

Reply:
i think there's been a miss in communication here - what i was originally a proponent for was to privatize as many fields as possible within the playerstats script, something i was advised against as members of the group wanted them public for ease of use in the editor.
i am still in favor of privatizing as much as possible and can certainly take care of that right quick, however i would like there to be a proper consensus on what is wanted public and what can be made private due to the playerstats script being an integral piece of interaction within the game. 
worth keeping in mind is that even if fields are privatized, it is no great challenge to implement methods to alter their values within the class itself.

Reply:
resolved during meeting to satisfaction of... probably everyone, i'd hope!